shared/src/proxies/getChiefJudgeNameForSigningProxy.js

```diff
@@ -0,0 +1,15 @@
+const {
+  getFeatureFlagValueInteractor,
+} = require('./featureFlag/getFeatureFlagValueProxy');
+
+/**
+ * getChiefJudgeNameForSigningInteractor
+ *
+ * @param {object} applicationContext the application context\
+ * @param {object} providers the providers object
+ * @returns {Promise<*>} the promise of the api call
+ */
+exports.getChiefJudgeNameForSigningInteractor = applicationContext =>
+  getFeatureFlagValueInteractor(applicationContext, {
```

Comment:
Shouldn't we get the `getFeatureFlagValueInteractor` from the `applicationContext` instead of importing it directly above?